### ISSUE TITLE
Picking The Game Resolution

### ISSUE THREAD

**Issue by SmugilouGazilou** [SmugilouGazilou/touhouemblem#23]
So. Pretty sure I made you all aware of this, but. UI Edits and such are going to be near to scarce without us deciding on the final resolution first.

The current size as of the 0.08a update would be 544x416.  0.09a is planned to have 680x480 resolution.

HOWEVER. I can easily adjust the resolution to up to 1920x1080.  So the real issue is.  I made all the UIs only fit for what the basic restrictions would be. Since you have to force the other resolution into play.

Simply put.  I want you all to help me decide on a **finalized resolution** so that I can set up **all future UI changes and features** to be accustomed to that as well as any other things.

Alternatively.

**You can give me two resolutions and I'll add those on top of being able to choose between 544x416 and 680x480** so that more players can get a beneficial experience regardless of their computer.

**Comment by crazycarl** [comment]
If you plan to use a full screen mode, default to the platform's native resolution. Making display mode changes is a surprisingly hairy operation. It can give you problems with alt+tabbing, mess up the desktop icons, and so forth. Don't even talk about what happens if two applications want to change the display mode.

If you are using windowed mode, 640x480 is going to be tiny, especially for users with a 4k monitor. That thing will be close in size of a desktop icon.

My suggestion is to just make the application use the native resolution for the window, and then emulate your desired resolution. This is actually what Minecraft does for it's UI, so you can look there if you want something to compare to. You'll need to make the UI adjust for different aspect ratios, but aside from that, it should generally look the same on all displays.
